tests/acceptance/CalendarModuleCest.php

```diff
@@ -0,0 +1,31 @@
+<?php 
+
+class CalendarModuleCest
+{
+    public function _before(AcceptanceTester $I)
+    {
+        $I->amOnPage('/index.php');
```

Comment:
This code should be abstracted as a Page object and be called from all other Cests